ansible/home/ec2-user/configure_webservers.yml

```diff
@@ -0,0 +1,68 @@
+- name: Todo Web Servers
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+  - name: trello_api_key
+    prompt: What is your Trello API Key?
+
+  - name: trello_api_token
+    prompt: What is your Trello API Token?
+
+  tasks:
+  - name: Install Git
+    ansible.builtin.yum:
+      name: git
+      state: latest
+    become: yes
+
+  - name: Install Python 3.11
+    ansible.builtin.yum:
+      name: python311
+      state: latest
+    become: yes
+
+  - name: Install Python Poetry
+    ansible.builtin.shell: curl -sSL https://install.python-poetry.org | python3 -
+    args:
+      creates: /home/ec2-user/.local/bin/poetry
+
+  - name: Create WebApp Folder
+    ansible.builtin.shell: mkdir /opt/todoapp
+    args:
+      creates: /opt/todoapp
+    become: yes
+
+  - name: Set WebApp Folder permissions
+    ansible.builtin.shell: chown ec2-user /opt/todoapp
+    become: yes
```

Comment:
Nothing wrong with these, but worth noting that e.g. you had to explicitly write it to be idempotent by adding the `creates` section, whereas if we took advantage of [the `file` module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html) we could let them handle that case for us:
```yml
- name: Create WebApp Folder
  ansible.builtin.file:
    path: /opt/todoapp
    state: directory
    owner: ec2-user
  become: yes